Symfony/CS/Fixer/Contrib/ClassElementsOrderFixer.php

```diff
@@ -0,0 +1,305 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Fixer\Contrib;
+
+use Symfony\CS\AbstractFixer;
+use Symfony\CS\Tokenizer\Token;
+use Symfony\CS\Tokenizer\Tokens;
+
+/**
+ * @author Gregor Harlan <gharlan@web.de>
+ */
+final class ClassElementsOrderFixer extends AbstractFixer
+{
+    private static $types = array(
```

Comment:
I will add a docblock.

The keys are all possible configuration elements. Value is an array of parent keys. So if a key is missing in configuration, the elements will be handled like the parent ones.